src/status/libstatus/settings.nim

```diff
@@ -65,4 +65,7 @@ proc getCurrentNetwork*(): Network =
 proc getCurrentNetworkDetails*(): NetworkDetails =
   let currNetwork = getSetting[string](Setting.Networks_CurrentNetwork, constants.DEFAULT_NETWORK_NAME)
   let networks = getSetting[seq[NetworkDetails]](Setting.Networks_Networks)
-  networks.find((network: NetworkDetails) => network.id == currNetwork)
\ No newline at end of file
+  networks.find((network: NetworkDetails) => network.id == currNetwork)
+    
+proc getLinkPreviewWhitelist*(): JsonNode =
+  result = callPrivateRPC("getLinkPreviewWhitelist".prefix, %* []).parseJSON()["result"]
```

Comment:
Is it possible that `getLinkPreviewWhitelist` fails?

Reply:
Ah, probably. I'll add a try/catch when I work on the design